### ISSUE TITLE
Ability to Focus next AdvanceEntry

### ISSUE THREAD

**Issue by LeoJHarris** [enisn/Xamarin.Forms.InputKit#116]
Hi there, 

Regarding the AdvancedEntry this is a feature request to be able to declare a "Next" AdvancedEntry that if the keyboard has the Return Key Type 'Next' pressed in the device keyboard it will put focus to the next AdvancedEntry, I could potentially add this feature if you would like it / think it might be useful? If the project doesn't already have that.

**Comment by enisn** [comment]
@LeoJHarris 
Yeah, currently it skips the next AdvancedEntry in same context. But keyboard closing and re-openning because button is return. I'll make it **Next** button instead of Return to ignore closing keyboard.

**Comment by LeoJHarris** [comment]
@enisn That'd be great because if there are next AdvancedEntry in the same context show the next icon then should prob display the next icon vs done, it would actually be nice to be able to manually set the return type key, as there are other beyond done and next.. i.e. search etc

**Comment by jqueiroz1** [comment]
Hi is there a way to prevent it from jumping to the next AdvancedEntry and be able to just close the keyboard?(Using Xamarin forms)
